ctapipe/io/astropy_helpers.py

```diff
@@ -21,7 +21,7 @@
 __all__ = ["h5_table_to_astropy"]
 
 
-def h5_table_to_astropy(h5file, path) -> QTable:
+def read_table(h5file, path) -> Table:
     """Get a table from a ctapipe-format HDF5 table as an `astropy.table.QTable`
```

Comment:
Probably a picky comment:
technically now this docstring is not completely correct, since what we really return now is a **Table** with the units stored in attributes as metadata

Reply:
This is not a picky comment but a good catch of lying documentation.

Reply:
fixed